tools/src/vcf2genomicsdb.cc

```diff
@@ -22,6 +22,7 @@
 
 #include "vcf2binary.h"
```

Comment:
Same comment as `gt_mpi_gather`? Do we need to overload vcf2genomicsdb with transcriptomic data?